core/frontend/src/tile/map/MapLayerSources.ts

```diff
@@ -225,14 +201,6 @@ export class MapLayerSources {
       (await ArcGisUtilities.getSourcesFromQuery(sourceRange)).forEach((queriedSource) => addSource(queriedSource));
     }
 
-    if (iModel && iModel.iTwinId && iModel.iModelId) {
```

Comment:
I think there's still a gap here.  
The widget loads previously saved layer definitions by calling `MapLayerSources.create` (MapLayerManager:186).  Now because `MapLayerSources.create` no longer includes layer definitions from Setting service, no layers will be loaded in the widget anymore.

Somehow, MapLayerSource objects will need to be created from `MapLayersUI,_iTwinConfig` and get injected into MapLayerManager's source list to keeps things working as previously.

Reply:
Sounds good, just updated the place where we call `MapLayerSources.create` to add the sources available using the `MapLayerPreferences`, which is the wrapper around the `MapLayersUI._itwinConfig`.

There's still some clean up needed I think to straighten some of the API out but it should be functional the same and all the other APIs are internal.

I'll add a test as well to validate.